tests/led/bloodlight_based.c

```diff
@@ -0,0 +1,41 @@
+#include <stddef.h>
+#include <stdint.h>
+#include <sys/printk.h>
+#include <sys/util.h>
+#include <string.h>
+#include <usb/usb_device.h>
+#include <drivers/uart.h>
+
+#include "../../src/led.h"
+#include "../../src/led.c"
```

Comment:
The project should have 'led.c' in the target_sources() in the CMakeLists.txt file and there is not need to include the sources like this. Also 'led.c' will have an include for 'led.h' and you wouldn't need two includes if you to include the 'led.c' source like this.

Reply:
why is it that by including led.c in target_sources() I don need to include it in  bloodlight_based.c?